config.yml

```diff
@@ -92,7 +95,41 @@ bulk_data_jwks:
       kid: b41528b6f37a9500edb8a905a595bdd7
 
     
-# preset fhir servers: optional. Minimally requires name, uri, module, optional inferno_uri, client_id, client_secret, scopes, instructions link
+# Optional preset server testing configurations.
+#
+# Universal Settings:
+# - name: REQUIRED. Name to be displayed in the preset selection menu.
+# - uri: REQUIRED. FHIR server base uri.
+# - module: REQUIRED. Name of testing module for this server.
+# - inferno_uri: OPTIONAL When this value is present, the preset will only be
+#   listed when inferno is being served from this uri. This allows separate
+#   SMART credentials to be used depending on where inferno is hosted. Omit this
+#   value to always display this preset.
+#
+# Single Patient API Settings (OPTIONAL):
+# - confidential_client: "true" for confidential clients, "false" for public
+#   clients. Used for SMART tests
+# - client_id: SMART client ID.
+# - client_secret: SMART client secret. Only used for confidential clients.
+# - instructions: Link to instructions for using this server.
+#
+# Multi Patient API Settings (OPTIONAL):
+# - bulk_url: Multi patient FHIR server base url.
+# - bulk_requires_auth: Whether the server requires SMART backend servics
+#   authorization. "true" or "false"
+# - bulk_token_endpoint: Token endpoint for SMART backend services
+#   authorization.
+# - bulk_client_id: Client ID for SMART backend services authorization.
+# - bulk_public_key: Public key for SMART backend services authorization.
+# - bulk_private_key: Private key for SMART backend services authorization.
+# - bulk_fastest_resource: A FHIR resource type accessible through the patient
+#   and group level exports that the server can export quickly.
+# - bulk_system_export_endpoint: Relative path to system export endpoint from
+#   'bulk_url'.
+# - bulk_patient_export_endpoint: Relative path to patient export endpoint from
+#   'bulk_url'.
+# - bulk_group_export_endpoint: Relative path to group export endpoint from
+#   'bulk_url'.
```

Comment:
Bulk data program testing uses subset of bulk configuration but it has two additional settings:
- bulk_scope
- group_id

Reply:
Ok, I added some descriptions. Let me know if they sound reasonable.

Reply:
The new addition looks good. One more question: Bulkdata test in Program does not use all the community settings. Ex, Program edition tests group export only so bulk_system_export_endpoint, bulk_patient_export_endpoint, and bulk_group_export_endpoint are not used at all. I think it is good to keep the file consistent between program and community. I also think the file should match what settings are actually relevant. So, I will leave the decision to you.

Reply:
I think simply sticking to consistency is fine.  I think this does highlight another place where we are putting test-specific stuff in a shared file, which is tough to manage.